tests/Microsoft.Bot.Builder.TestBot.Json/Samples/RandomGenerator/RandomGenerator.main.dialog

```diff
@@ -0,0 +1,37 @@
+﻿{
+    "$schema": "../../testbot.schema",
+    "$kind": "Microsoft.AdaptiveDialog",
+    "triggers": [
+        {
+            "$kind": "Microsoft.OnBeginDialog",
+          "actions": [
+            {
+              "$kind": "Microsoft.EmitEvent",
+              "eventName": "setTestOptions",
+              "EventValue": {
+                "randomSeed": 100
+              }
```

Comment:
NO, this is wrong direction and wrong eventing.

The testScript needs to send a EventActivity to the BOT, not the BOT=> emitting a dialog event to the stack.